templates/index.html

```diff
@@ -5,8 +5,9 @@
     <div class="container">
       <div class="row mt-3">
         <div class="col-md-4 mx-auto">
+		  <center><span><i><h4>All fields marked with * are mandatory</h4></i></span></center><br>
```

Comment:
No need for a help text.

Reply:
But I was thinking it would be nice to have it. Might help the user know what the * refers to. Not everyone might know what the significance of the * is. Just my opinion though. Would remove it if you want me to though

Reply:
+1